File: v_1.0.2/source/hin-web/src/main/webapp/html/js/vo/ChatContent.js
var HIN;
if (!HIN)
	HIN = {};

HIN.ChatContent = function() {
	this.sender = null;
	this.receiver = null;
	this.content = null;
	this.sendTime = null;
	this.readTime = null;
	this.readStatus = null;
	this.saveType = null;
	this.instanceId = null;
	chatContent = this;
};
HIN.ChatContent.prototype.toString = function() {
	return "[" + this.instanceId + " , " + this.sender + " , " + this.receiver + " , " + this.content
			+ " , " + this.readStatus + " , " + this.saveType +	"]";
};

File: v_1.0.2/source/hin-common/src/main/java/com/hin/domain/ChatContent.java
package com.hin.domain;

public class ChatContent {

	private String sender;
	private String receiver;
	private String content;
	private String sendTime;
	private String readTime;
	private String readStatus;
	private String saveType="both";
	private String instanceId;
	private String organizationId;

	public String getSaveType() {
		return saveType;
	}

	public void setSaveType(String saveType) {
		this.saveType = saveType;
	}

	public String getInstanceId() {
		return instanceId;
	}

	public void setInstanceId(String instanceId) {
		this.instanceId = instanceId;
	}

	public String getSender() {
		return sender;
	}

	public void setSender(String sender) {
		this.sender = sender;
	}

	public String getReceiver() {
		return receiver;
	}

	public void setReceiver(String receiver) {
		this.receiver = receiver;
	}

	public String getContent() {
		return content;
	}

	public void setContent(String content) {
		this.content = content;
	}

	public String getSendTime() {
		return sendTime;
	}

	public void setSendTime(String sendTime) {
		this.sendTime = sendTime;
	}

	public String getReadTime() {
		return readTime;
	}

	public void setReadTime(String readTime) {
		this.readTime = readTime;
	}

	public String getReadStatus() {
		return readStatus;
	}

	public void setReadStatus(String readStatus) {
		this.readStatus = readStatus;
	}

	/**
	 * @return the organizationId
	 */
	public String getOrganizationId() {
		return organizationId;
	}

	/**
	 * @param organizationId the organizationId to set
	 */
	public void setOrganizationId(String organizationId) {
		this.organizationId = organizationId;
	}
}
